src/Exercism.Analyzers.CSharp/Analyzers/Shared/SharedComments.cs

```diff
@@ -34,5 +34,8 @@ internal static class SharedComments
 
         public static SolutionComment InvalidMethodSignature(string method, string signature) =>
             new SolutionComment("csharp.general.invalid_method_signature", new SolutionCommentParameter(Name, method), new SolutionCommentParameter(Signature, signature));
+
+        public static SolutionComment MissingProperty(string property) =>
+            new("csharp.general.missing_property", new SolutionCommentParameter(Name, property));
```

Comment:
FYI For each key of a property, an entry must be added to the corresponding website-copy directory: https://github.com/exercism/website-copy/tree/main/analyzer-comments/csharp